src/components/plugins/AddPluginDialog.tsx

```diff
@@ -53,9 +52,9 @@ const useStyles = makeStyles({
   whiteButton: {
     textTransform: "none",
     backgroundColor: "transparent",
-    borderColor: `${middleGrey} !important`,
+    borderColor: `${lightGrey} !important`,
```

Comment:
Should we still have any makeStyles here @ChrisBaidoo - I would have thought this would be in STYLE now?